code/modules/mob/living/living_defines.dm

```diff
@@ -68,3 +68,5 @@
 
 	var/mob/living/mental_dominator //The person controlling the mind of this person, if applicable
 	var/mob/living/mind_control_holder/mind_control_holder //If the mob is being mind controlled, where their old mind is stored (check clock_mobs.dm)
+
+	var/obj/effect/proc_holder/ranged_ability //Any ranged ability the mob has, as a click override

```

Comment:
Humans can use it, and I plan to port other things, such as the Fireball spell to it, which any /living mob can use. It is built for the future!